File: app/src/main/java/com/jy/meeting/common/TestActivity.java
package com.jy.meeting.common;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.ximalife.library.view.FlowLayout;

public class TestActivity extends AppCompatActivity {



    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        FlowLayout.ItemView<String> itemView = new FlowLayout.ItemView<String>() {
            @Override
            public void getCover(String item, FlowLayout.ViewHolder holder, View inflate, int position) {

            }
        };
    }
}

File: app/src/main/java/com/jy/meeting/common/TestActivity.kt
package com.jy.meeting.common

import com.jy.meeting.R
import com.jy.meeting.databinding.ActivityTestBinding
import com.jy.meeting.view.DrawCircle
import com.ximalife.library.base.BaseActivity


class TestActivity : BaseActivity<ActivityTestBinding>(ActivityTestBinding::inflate) {

    override fun initView() {

        var circle = findViewById<DrawCircle>(R.id.circle)

//        circle.setZoomEnabled(true);
//        circle.getZoomer()!!.setReadMode(false);
//        circle.getZoomer()!!.zoom(3f, true)
//        circle.displayResourceImage(R.mipmap.ic_guid_bg)

    }

}
